dev_docs/key_concepts/encrypted_saved_objects.mdx

```diff
@@ -0,0 +1,175 @@
+---
+id: kibDevDocsEncryptedSavedObjectsIntro
+slug: /kibana-dev-docs/key-concepts/encrypted-saved-objects-intro
+title: Encrypted Saved Objects
+description: Configure your saved object types to secure sensitive data.
+date: 2024-05-21
+tags: ['kibana', 'dev', 'contributor', 'api docs']
+---
+
+"Encrypted Saved Objects" (ESO) are <DocLink id="kibDevDocsSavedObjectsIntro" text="Saved Object types" /> that have been registered with the Encrypted Saved Objects
+Service (ESO Service) to the specify which attributes should be protected (encrypted attributes) and which attributes, if any, should be present and unchanged
+in order to decrypt the protected attributes ("Additional Authenticated Data", or AAD).
```

Comment:
QQ: I think for me as the least familiar person with the definition of AAD it is a bit unclear. Does AAD term cover both encrypted attributes and the ones used for decryption or ONLY attributes used for decryption?